glossary.yml

```diff
@@ -2086,7 +2086,7 @@
     acronym: "DOI"
     def: >
       A unique persistent identifier for a book, paper, report, dataset, software release, or
-      other digital artefact.
+      other digital artifact.
```

Comment:
@fmichonneau 

You told me British spellings are preferred. Can you confirm this, in which case this change (and at least one other here, to `stylise`) should not be made.

Reply:
yes within The Carpentries we favor British spelling

Reply:
```suggestion
      other digital artefact.
```